Sources/SwifterSwift/Foundation/DateExtensions.swift

```diff
@@ -487,6 +487,20 @@ public extension Date {
         return timeIntervalSince1970
     }
 
+    /// SwifterSwift: Current day in a year
+    var dayInYear: Int {
+        return calendar.ordinality(of: .day, in: .year, for: self)!
+    }
+
+    /// SwifterSwift: How many days in a year
+    var numberOfDaysInYear: Int {
+        return (calendar.range(of: .day, in: .year, for: self)!.upperBound) - 1
```

Comment:
Same question about nil here too :)

Reply:
Same response as the previous.
smaller calendar component, day, is always smaller than larger calendar component year.
If I understood correctly it won't be nil by any chance :)